x-pack/test/api_integration/apis/alerting/alerts.ts

```diff
@@ -139,5 +139,50 @@ export default function alertTests({ getService }: KibanaFunctionalTestDefaultPr
         source: 'action:test.index-record',
       });
     });
+
+    it('should retry failures', async () => {
```

Comment:
Not too sure about this test.  It will take a long time to run I think, and it is also really just testing behavior of the task manager, isn't it?

Reply:
This will depend how the conversation goes next week, the test is designed to assume task manager retries right away. Currently the task manager doesn't, it waits 5m to retry so I have to skip test for now (added TODO in PR description). If ever we don't want to retry right away (based on conversation next week), we'll delete it.